contracts/upgradeable_contracts/RelativeDailyLimit.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity 0.4.24;
+
+import "./BaseRelativeDailyLimit.sol";
+
+contract RelativeDailyLimit is BaseRelativeDailyLimit {
+    function _minPerTx() internal view returns (uint256) {
+        return minPerTx();
+    }
+
+    function dailyLimit() public view returns (uint256) {
+        return _getTodayLimit();
+    }
+
+    function setMinPerTx(uint256 _minPerTx) external onlyOwner {
+        require(_minPerTx < maxPerTx());
+        uintStorage[MIN_PER_TX] = _minPerTx;
+    }
+
+    function setMaxPerTx(uint256 _maxPerTx) external onlyOwner {
+        require(_maxPerTx > minPerTx());
+        uintStorage[MAX_PER_TX] = _maxPerTx;
+    }
```

Comment:
In the current implementation when we need to stop bridge operations (for example in case for xDai bridge upgrade) we use `setMaxPerTx` to set this parameter to `0`. In this method implementation, while the check `require(_maxPerTx > minPerTx());` does makes sense for the integrity of the parameters we will be loosing the ability to stop the bridge operations. @akolotov  does it makes sense to remove this check in order to be able to stop bridge operations?

Reply:
Are you about `setDalyLimit` method? https://github.com/poanetwork/tokenbridge-contracts/blob/develop/contracts/upgradeable_contracts/BasicTokenBridge.sol#L75

We can make the same check for this implementation of `setMaxPerTx`:
```
require(_maxPerTx > minPerTx() || _maxPerTx == 0);
```

Reply:
Yes, I think having that check it would work